.github/CODEOWNERS

```diff
@@ -12,10 +12,11 @@
 /src/libs/ajax/Catalog.ts @DataBiosphere/data-explorer-eng
 /src/libs/ajax/workflows-app/ @DataBiosphere/broad-workflow-management @DataBiosphere/broad-workflow-execution
 /src/pages/library/DataBrowser* @DataBiosphere/data-explorer-eng
-/src/pages/library/dataBrowser-utils* @DataBiosphere/data-explorer-eng
 /src/pages/library/Datasets* @DataBiosphere/data-explorer-eng
 /src/pages/library/SearchAndFilterComponent* @DataBiosphere/data-explorer-eng
 /src/pages/library/test-datasets.ts @DataBiosphere/data-explorer-eng
```

Comment:
I don't see this path either, or `/src/constants/datasets.js` above

Reply:
There is still one at `/src/constants/datasets.js`, however test-datasets is stale and I can delete that.